console/src/view/mod.rs

```diff
@@ -20,18 +31,28 @@ pub struct View {
     /// details view), we want to leave the task list's state the way we left it
     /// --- e.g., if the user previously selected a particular sorting, we want
     /// it to remain sorted that way when we return to it.
-    list: tasks::List,
+    tasks_list: TableListState<Task, TasksSort>,
+    resources_list: TableListState<Resource, ResourcesSort>,
     state: ViewState,
     pub(crate) styles: Styles,
 }
 
 enum ViewState {
     /// The table list of all tasks.
     TasksList,
+    /// The table list of all resources.
+    ResourcesList,
     /// Inspecting a single task instance.
     TaskInstance(self::task::TaskView),
 }
 
+#[derive(Eq, PartialEq)]
+pub(crate) enum CurrentView {
+    Tasks,
+    Resources,
+    Task,
+}
```

Comment:
I'm a bit on the fence about adding a `CurrentView` enum in addition to `ViewState`.Why not just change `ViewState` like this:
```suggestion
#[derive(Eq, PartialEq)]
pub(crate) enum ViewState {
    /// The table list of all tasks.
    TasksList,
    /// The table list of all resources.
    ResourcesList,
    /// Inspecting a single task instance.
    TaskInstance(self::task::TaskView),
}
```
and have the `current_view` method be
```rust
    pub(crate) current_view(&self) -> &ViewState
        &self.state
    }
```

Not a big deal, but it might be simpler...